palisade-service/src/unit-tests/java/uk/gov/gchq/palisade/service/palisade/domain/DataRequestTest.java

```diff
@@ -63,9 +62,11 @@ public void storeAndRetrieveTest() {
         this.dataRequestRepository.save(entity);
         final DataRequestEntity subject = this.dataRequestRepository.getByRequestId("identifier-x");
 
-        assertThat("The user id is not preserved through persistence", subject.getUser().getUserId().getId(), is(equalTo("archibald")));
-        assertThat("The request id is not preserved through persistence", subject.getRequestId(), is(equalTo("identifier-x")));
-        assertThat("The context is not preserved through persistence", subject.getContext().get("testing repo"), is(equalTo("this")));
+        assertAll(
```

Comment:
No need for the assertAll. Just use the 3 asserts.

Reply:
The `assertAll` has been removed